examples/jupyter/mg-mode-part-i.ipynb

```diff
@@ -205,7 +197,16 @@
    "cell_type": "code",
    "execution_count": 7,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/home/shriwise/opt/openmc/openmc/openmc/surface.py:1511: FutureWarning: \"ZCylinder(...) accepts an argument named 'r', not 'R'. Future versions of OpenMC will not accept the capitalized version.\n",
+      "  FutureWarning)\n"
```

Comment:
While you're at it, do you mind addressing this?

Reply:
Can do!